CONTRIBUTING.md

```diff
@@ -2,6 +2,11 @@
 
 ## Getting Started
 
+### Code of Conduct
+
+
+You can find our code of conduct at [CODE_OF_CONDUCT.md][https://github.com/OpenMined/Grid/CODE_OF_CONDUCT.md]. We ask that all members of the community follow this code in order to promote a welcoming and productive environment.
```

Comment:
Replace: [CODE_OF_CONDUCT.md][https://github.com/OpenMined/Grid/CODE_OF_CONDUCT.md] to `[CODE_OF_CONDUCT.md](https://github.com/OpenMined/Grid/CODE_OF_CONDUCT.md)`